condensedinlinepanel/static/condensedinlinepanel/src/components/CardSet.tsx

```diff
@@ -178,7 +179,7 @@ export class CardSet extends React.Component<CardSetProps, {}> {
                 e.preventDefault();
                 return false;
             };
-            addButton = <button className="condensed-inline-panel__top-add-button button bicolor icon icon-plus" type="button" onClick={onClickAddButton}>Add</button>;
+            addButton = <button className="condensed-inline-panel__top-add-button button bicolor icon icon-plus" type="button" onClick={onClickAddButton}>Add {this.props.panelLabel}</button>;
```

Comment:
I'm thinking it might be nicer to make the ``panelLabel`` argument replace the whole label, including "Add". This would be useful for people working in languages that might put the word for "Add" after the label.